utils/src/main/kotlin/CopyrightStatementsProcessor.kt

```diff
@@ -79,6 +79,130 @@ private fun prettyPrintYears(years: Collection<Int>) =
         if (fromYear == toYear) fromYear.toString() else "$fromYear-$toYear"
     }
 
+private fun determineParts(copyrightStatement: String): Parts? {
```

Comment:
Commit message:

> are better places outside

are better place*d* outside

Reply:
Also, there seems to be a *detekt* issue.

Reply:
Done.